src/components/LyricView.tsx

```diff
@@ -130,6 +138,11 @@ const LyricView: React.FC<LyricViewProps> = (props: LyricViewProps) => {
       </Fragment>
     );
   }
+
+  function ToggleNavButtonListener() {
```

Comment:
I think this function should be in SongPage.tsx, and dictate only whether or not to show the left/right buttons. From what I see in the code LyricView doesn't change regardless of tooNarrow becoming true

Reply:
Yea if we are going to detect overlapping then the function should live in songpage tsx. The reason it lives in lyricview right now is because song view doesn't need this listener imo (you need to shrink the page to like 550 pixel on song view for the buttons to overlap).  

> From what I see in the code LyricView doesn't change regardless of tooNarrow becoming true

really? if `tooNarrow` is true in lyric view, the buttons shouldn't be getting rendered.